Libs/Python/ShapeworksPython.cpp

```diff
@@ -1204,7 +1254,7 @@ PYBIND11_MODULE(shapeworks_py, m)
 
   //TODO: See github issue #966
   .def("setField",
-       [](Mesh &mesh, std::vector<double>& v, std::string name) -> decltype(auto) {
+       [](Mesh &mesh, std::string name, std::vector<double>& v) -> decltype(auto) {
```

Comment:
Please change the GitHub issue to #1495. Also on line 1268 and anywhere else it's mentioned in this file. Details for why are in a previous comment. Thanks!

Reply:
consolidated the many places this was duplicated into a helper function and added a comment there